modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/redshift/ShredModel.scala

```diff
@@ -0,0 +1,156 @@
+package com.snowplowanalytics.iglu.schemaddl.redshift
+
+import cats.data.NonEmptyList
+import cats.syntax.parallel._
+import cats.syntax.show._
+import cats.syntax.either._
+import io.circe.Json
+import com.snowplowanalytics.iglu.core.SchemaKey
+import com.snowplowanalytics.iglu.schemaddl.StringUtils.snakeCase
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.{Pointer, Schema}
+import com.snowplowanalytics.iglu.schemaddl.redshift.internal.{FlatSchema, Migrations, ShredModelEntry}
+import com.snowplowanalytics.iglu.schemaddl.redshift.internal.ShredModelEntry.ColumnType
+import com.snowplowanalytics.iglu.schemaddl.redshift.internal.ShredModelEntry.CompressionEncoding.Text255Encoding
+import com.snowplowanalytics.iglu.schemaddl.redshift.internal.Migrations._
+
+import math.abs
+
+/**
+ *
+ * @param entries    - list of model entries, containing the schema pointers and bottom level sub schemas 
+ * @param schemaKey  - schema key of corresponding top level schema 
+ * @param isRecovery - whether or not this model is used for automatic recovery. Only affects the SQL representation.
+ * @param migrations - migrations accumulated though merging with next schemas in family.
+ */
+case class ShredModel(
+                       entries: List[ShredModelEntry],
```

Comment:
`entries` is publicly visible on this class, but `ShredModelEntry` is in the `internal` package.  Does that matter?

Reply:
should be private.